App.Stocks/Services/StocksManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using App.Configuration;
+using App.Stocks.Interfaces;
+using App.Stocks.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace App.Stocks.Services
+{
+    public class StocksManager : IStocksManager, ITransientDependency
+    {
+        private  ICompaniesRepository repository;
+
+        private  IValidateServices validateServices;
+
+        public StocksManager(ICompaniesRepository repository,IValidateServices validateServices)
+        {
+            this.validateServices = validateServices;
+            this.repository = repository;
+        }
+
+        public async Task<IEnumerable<StockView>> CompanyStocks(int companyId)
+        {
+
+            var company = await Task.Run(() => repository.CompanyById(companyId));
```

Comment:
method "CompanyById" (whic should be renamed) is already asynchronous and dont need to be wrapped with Task.Run